### ISSUE TITLE
Merge prefix error

### ISSUE THREAD

**Issue by yisibl** [clean-css/clean-css#700]
## Input

``` css
.foo {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}
.bar {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: auto;
}

.baz {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}
```
## Output

``` css
.bar,
.foo {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox
}

.foo {
  display: flex
}

.bar {
  display: flex;
  margin: auto
}

.baz {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex
}
```

:joy: :joy: :joy: 
## Expect

``` css
.foo, .bar, .baz {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex
}

.bar {
  margin: auto
}
```

Used `gulp-minify-css` 1.2.1.

**Comment by jakubpawlowicz** [comment]
Multi-property restructuring is still not ideal, however as far as I can see it still delivers a correct CSS, right?

**Comment by yisibl** [comment]
If this feature is not perfect, I hope to be able to disable the default.

**Comment by jakubpawlowicz** [comment]
The next major version will change the defaults making restructuring disabled by default. You can still do it now via `--skip-restructuring` switch.

**Comment by yisibl** [comment]
Thank you very much, I hope the CSS compression tool is always safe.

**Comment by jakubpawlowicz** [comment]
Depends on #623.

**Comment by jakubpawlowicz** [comment]
Won't fix as clean-css is going into [maintenance mode](https://github.com/clean-css/clean-css/discussions/1209).
